slides/intro-02-data-budget.qmd

```diff
@@ -37,33 +37,33 @@ knitr:
 :::
 
 ::: {.column width="40%"}
-![](images/taxi_spinning.svg)
+![](images/forest_mountain.svg)
 :::
 
 :::
 
 ::: footer
-Credit: <https://www.svgrepo.com/svg/8322/taxi>
+Credit: <https://www.svgrepo.com/svg/251793/forest-mountain>
 :::
 
 ## Data on forests in Washington
 
 ::: columns
-::: {.column width="60%"}
+::: {.column width="70%"}
 -   `N = 7,109` plots of land, one from each of 7,109 6000-acre hexagons in WA.
 -   A nominal outcome, `forested`, with levels `"Yes"` and `"No"`, measured "on-the-ground."
 -   18 remotely-sensed and easily-accessible predictors:
      - **numeric** variables based on weather and topography.
      - **nominal** variables based on classifications from other governmental orgs.
 :::
 
-::: {.column width="40%"}
-![](images/taxi.png)
+::: {.column width="30%"}
```

Comment:
This was needed to ensure that all the text in the other column fits on the slide.